docs/en/roadmap/Environment-Setup.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: global
+title: Environment Setup
+group: Roadmap
```

Comment:
the group name must match one of the category names so this needs to be `Contribution Roadmap` and same for all the other files

Reply:
Changed, thanks!